pyclesperanto_prototype/_tier1/_generate_touch_matrix.py

```diff
@@ -6,8 +6,28 @@
 @plugin_function(output_creator=create_square_matrix_from_labelmap)
 def generate_touch_matrix(labelmap :Image, touch_matrix_output :Image = None):
     from .._tier1 import set
-    """
-    docs
+    """Takes a labelmap with n labels and generates a (n+1)*(n+1) matrix where all pixels are set to 0 exept those where labels are touching. 
```

Comment:
Cool! This is also called a Region Adjacency Graph, might want to reference that term somehow.